src/components/testCaseLanding/common/runTestsButton/RunTestsButton.tsx

```diff
@@ -22,6 +34,26 @@ export default function RunTestButton(props: RunTestButtonProps) {
     }, 500);
   }
 
+  useEffect(() => {
+    if (
+      measure?.model === Model.QDM_5_6 &&
+      featureFlags?.disableRunTestCaseWithObservStrat
+    ) {
+      const groups: Group[] = measure?.groups;
+      groups?.forEach((group) => {
+        const measureObservations: MeasureObservation[] =
+          group?.measureObservations;
+        const measureStratifications: Stratification[] = group?.stratifications;
+        if (
+          measureObservations ||
```

Comment:
Can we also do the length check on observations as well? Not sure if they would add observations and then remove them if we might end up with an empty observations array which would then disable the button

Reply:
@nmorasb done.